### PR TITLE
Add showSixWeeks option

### PR DESCRIPTION
**Pull request by TomCorvus**
Add an option to always display 6 weeks on each month. https://github.com/wix/react-native-calendars/issues/1073

Need to add:

`    /**
     *  Always show six weeks on each month. Default = false
     */
    showSixWeeks?: boolean; `

in node_modules/@types/react-native-calendars/index.d.ts

test

### CODE REVIEW

File: src/calendar/index.js [hunk 1]
```diff
@@ -287,7 +289,8 @@ class Calendar extends Component {
   }
 
   render() {
-    const days = dateutils.page(this.state.currentMonth, this.props.firstDay);
+    const days = dateutils.page(this.state.currentMonth, this.props.firstDay, this.props.showSixWeeks, this.props.hideExtraDays);
```
Comment (chenei) [inline src/calendar/index.js:292]: I have a suggestion here:
```
const {currentMonth} = this.state;
const {firstDay, showSixWeeks, hideExtraDays} = this.props;
const shouldShowSixWeeks = showSixWeeks && !hideExtraDays;
const days = dateutils.page(currentMonth, firstDay, shouldShowSixWeeks);
```

File: src/dateutils.js [hunk 2]
```diff
@@ -69,6 +71,12 @@ function page(xd, firstDayOfWeek) {
     to.addDays((ldow + 7 - day) % 7);
   }
 
+  const daysForSixWeeks = (((daysBefore + days.length) / 6) >= 6);
+
+  if (showSixWeeks && !daysForSixWeeks && hideExtraDays !== true) {
```
Comment (chenei) [inline src/dateutils.js:76]: Please see my comment above:
```
if (showSixWeeks && !daysForSixWeeks) {
```

**Comment by TomCorvus** [comment]
Thanks. It's more readable. I pushed changes.

**Comment by chenei** [comment]
@TomCorvus just resolve the conflicts :)

**Comment by TomCorvus** [comment]
oops! It's done.

**Pull request merged**